src/igvfd/schemas/software.json

```diff
@@ -71,12 +71,12 @@
         }
     },
     "fuzzy_searchable_fields": [
-        "title",
         "@type",
+        "description",
```

Comment:
I think we agreed that description was a second class citizen and should not be searchable